### PR TITLE
bulk_update set updated_time

### PR DESCRIPTION
**Pull request by alex-smile**
### Description

Fixes # (issue)

### Checklist

- [ ] 填写 PR 描述及相关 issue (write PR description and related issue)
- [ ] 代码风格检查通过 (code style check passed)
- [ ] PR 中包含单元测试 (include unit test)
- [ ] 单元测试通过 (unit test passed)
- [ ] 本地开发联调环境验证通过 (local development environment verification passed)

### CODE REVIEW

**Review by wklken — APPROVED** [review]

File: src/dashboard/apigateway/apigateway/biz/plugin/plugin_synchronizers.py [hunk 1]
```diff
@@ -63,6 +64,7 @@ def sync(
                     plugin_config_obj = existing_binding.config
                     plugin_config_obj.config = plugin_config_data.yaml
                     plugin_config_obj.updated_by = username
+                    plugin_config_obj.updated_time = now_datetime()
```
Comment (zhu327) [inline src/dashboard/apigateway/apigateway/biz/plugin/plugin_synchronizers.py:67]: 同上

File: src/dashboard/apigateway/apigateway/biz/backend.py [hunk 2]
```diff
@@ -78,10 +79,11 @@ def update(backend: Backend, data: Dict[str, Any], updated_by: str) -> Backend:
 
             backend_config.config = new_config
             backend_config.updated_by = updated_by
+            backend_config.updated_time = now_datetime()
```
[the remark this answers is not in the document]
Comment (alex-smile) [reply src/dashboard/apigateway/apigateway/biz/backend.py]: 1

File: src/dashboard/apigateway/apigateway/biz/plugin/plugin_synchronizers.py (hunk 1 above)
Comment (alex-smile) [reply src/dashboard/apigateway/apigateway/biz/plugin/plugin_synchronizers.py]: 1

File: src/dashboard/apigateway/apigateway/biz/resource/savers.py [hunk 3]
```diff
@@ -74,6 +75,7 @@ def _save_resources(self) -> bool:
             if resource_data.resource:
                 resource = resource_data.resource
                 resource.updated_by = self.username
+                resource.updated_time = now_datetime()
```
[the remark this answers is not in the document]
Comment (alex-smile) [reply src/dashboard/apigateway/apigateway/biz/resource/savers.py]: 1

**Review by zhu327 — APPROVED** [review]

**Pull request merged**